CMakeSettings.json

```diff
@@ -0,0 +1,16 @@
+﻿{
```

Comment:
It seems this file addition was not intentional.

Reply:
No, it wasn't - removed it...